src/parser/parsers/heroes.py

```diff
@@ -202,3 +164,41 @@ def _map_attr_names(self, data, func):
             output_data[mapped_key] = data[key]
 
         return output_data
+    
+def calc_bullet_velocity(spline):
```

Comment:
I meant just change the name to `_calc_bullet_velocity`, not move it out of the class. It doesn't make much difference, but its good to keep it consistent with the rest of the functions